repo/config/init.go

```diff
@@ -40,7 +37,7 @@ func Init(out io.Writer, nBitsForKeypair int) (*Config, error) {
 			Gateway: "/ip4/127.0.0.1/tcp/8080",
 		},
 
-		Datastore: datastore,
+		Datastore: *datastore,
```

Comment:
seems like needless dereferencing. Probably either make this type a pointer, or make the `DefaultDatastoreConfig` method return a concrete object.